### PR TITLE
po: Add French translation

### PR DESCRIPTION
**Pull request by TheEvilSkeleton**
Closes https://github.com/flatpak/flatpak/issues/3437

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -0,0 +1,5489 @@
+# French translation for flatpak.
+# Copyright (C) 2021 Flatpak team and others.
+# This file is distributed under the same license as the flatpak package.
+# Hari Rana / TheEvilSkeleton <theevilskeleton@riseup.net>, 2021.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: flatpak 1.11.3\n"
```
Comment (AsciiWolf) [inline po/fr.po:8]: This should probably be `flatpak master\n` instead.

File: po/fr.po [hunk 2]
```diff
@@ -0,0 +1,5489 @@
...
+"Report-Msgid-Bugs-To: https://github.com/flatpak/flatpak/issues\n"
+"POT-Creation-Date: 2021-11-08 20:15-0600\n"
+"PO-Revision-Date: 2021-11-09 08:11-0500\n"
+"Last-Translator: FULL NAME <theevilskeleton@riseup.net>\n"
```
Comment (AsciiWolf) [inline po/fr.po:12]: "FULL NAME" should probably be replaced by your actual name.

Comment (TheEvilSkeleton) [reply po/fr.po:12]: Thanks. I would've sworn I put my name.

**Comment by AsciiWolf** [comment]
LGTM, I think this is good to be merged as it is now, it can be updated later. :-)

**Comment by TheEvilSkeleton** [comment]
I hardly translated! I was thinking of translating everything when I have time so I can and then set it as review. If this looks good for now then I'm just going to set it as review and translate later.

**Comment by mwleeds** [comment]
Rebased this on master

**Pull request merged**